src/game_events/action_wml.cpp

```diff
@@ -1684,7 +1684,10 @@ WML_HANDLER_FUNCTION(unstore_unit, /*event_info*/, cfg)
 			resources::units->erase(loc);
 			resources::units->add(loc, *u);
 
-			std::string text = cfg["text"];
+			config::attribute_value text = var["gender"].str() == "female" ? cfg["female_text"] : cfg["male_text"];
```

Comment:
I only want to fallback to text= if the gender-specific key is undefined, and not if it's defined as an empty string; for example:

```
[animate_unit]
  text=_"Stuff"
  female_text=""
[/animate_unit
```

That should show the text only if the unit is male.